packages/node_modules/@ciscospark/widget-space/src/container.js

```diff
@@ -249,6 +306,17 @@ export class SpaceWidget extends Component {
         </div>
       );
     }
+    else if (errors.get(`hasError`)) {
```

Comment:
Adding this error display here as well because we may get an error before we have a spark instance or space details.